### ISSUE TITLE
Add overloads for String type of chat_id

### ISSUE THREAD

**Issue by naftalmm** [kotlin-telegram-bot/kotlin-telegram-bot#128]
In order to interact with public channels via Bot API, `chat_id` should be a `String` (in the format `@channelusername`). Currently, only few methods have such overload variant (with `channelUsername` argument name for `chat_id`, which is a bit confusing on my taste, but still better than nothing):
- [x] sendMessage
- [ ] forwardMessage (need overloads for `chat_id` & `from_chat_id` parameters, so it should be 4 methods summary)
- [ ] copyMessage (need overloads for `chat_id` & `from_chat_id` parameters, so it should be 4 methods summary)
- [ ] sendPhoto
- [ ] sendAudio
- [ ] sendDocument
- [ ] sendVideo
- [ ] sendAnimation
- [ ] sendVoice
- [ ] sendVideoNote
- [ ] sendMediaGroup
- [ ] sendLocation
- [ ] editMessageLiveLocation
- [ ] stopMessageLiveLocation
- [ ] sendVenue
- [ ] sendContact
- [x] sendPoll
- [x] sendDice
- [ ] sendChatAction
- [ ] kickChatMember
- [ ] unbanChatMember
- [ ] restrictChatMember
- [ ] promoteChatMember
- [x] setChatAdministratorCustomTitle
- [ ] setChatPermissions
- [ ] exportChatInviteLink
- [ ] setChatPhoto
- [ ] deleteChatPhoto
- [ ] setChatTitle
- [ ] setChatDescription
- [ ] pinChatMessage
- [ ] unpinChatMessage
- [ ] unpinAllChatMessages
- [ ] leaveChat
- [ ] getChat
- [ ] getChatAdministrators
- [ ] getChatMembersCount
- [ ] getChatMember
- [ ] setChatStickerSet
- [ ] deleteChatStickerSet
- [ ] editMessageText
- [ ] editMessageCaption
- [ ] editMessageMedia
- [ ] editMessageReplyMarkup
- [ ] stopPoll
- [x] deleteMessage
- [ ] sendSticker

Bunch of copy-paste should do the thing. Also there are 2 implementation options:
1. Preserve old or choose new name for `chat_id` parameter with `String` type. It should be consistent across all these methods. I believe it should be `chatId`, but it'll be a breaking change. Why `channelUsername` name was chosen initially on the first place?
2. Auto-add `@`-symbol in the beginning of `chat_id` string if it's not starting with it
```kotlin
val _channelUsername = if (channelUsername.startsWith("@")) channelUsername else "@$channelUsername"
```
 or provide some check about it:
```kotlin
require(channelUsername.startsWith("@")) {
    "Channel name should be in the format @channelusername"
}
```
Personally, I'd prefer auto-add option.

**Comment by vjgarciag96** [comment]
Hi @naftalmm! Thanks for opening such a detailed issue. I totally agree with you, we should provide the version of those bot operations for channels too.

The reason to call the argument `channelUsername` is to avoid confusion and for clarity. A chat id and a channel username are different things, so calling a channel username `chatId` could be a bit annoying.  Also, if we call the channel username argument `chatId`, users of the API will probably think that they should use a chat id to interact with a channel rather than using a channel username.

Regarding the implementation options, I'd be happy with any of them 👌🏻 (probably the one with the auto-add option is more helpful).

**Comment by Flame239** [comment]
Hey 👋 
[In this branch](https://github.com/Flame239/kotlin-telegram-bot/compare/master...Flame239:copy-message-new-api) you will find my idea how to implement this. 
TL;DR:  there is brand new `ChatId` class which encapsulates either chatId or channel username and we can use it like 
```
chatId = ChatId.fromUsername("username")
or
chatId = ChatId.fromId(123)
```

Reasoning:
Copy pasting almost identical pieces of code (and doing this 4 times for a lot of methods) is just crazy 😱 , anyways all those identifiers will be converted to string. Also I decided not inventing new parameter names and stayed with just `chatId` as it is used in official documentation and anybody who will start working with this bot api will see clear 1-to-1 mapping between official docs and our api. I personally was a little confused at first with all those method overloads with different param names and it took me a while to understand what is the difference between them.

**Comment by vjgarciag96** [comment]
Hi @Flame239!

It looks good to me overall, I'm happy to go with that approach. If you open a PR with the changes I can leave my feedback in there 🙂

Cheers!

**Comment by Flame239** [comment]
@vjgarciag96 please check this pr: https://github.com/kotlin-telegram-bot/kotlin-telegram-bot/pull/170/files

**Comment by Flame239** [comment]
Closed in https://github.com/kotlin-telegram-bot/kotlin-telegram-bot/pull/172
